tests/test_conan_version_comparison.py

```diff
@@ -0,0 +1,113 @@
+import pytest
```

Comment:
This file likely also comes from conan and likely therefore need a notice header and ABOUT file?

Reply:
All the tests that are copied comes from https://github.com/conan-io/conan/tree/release/2.0/conans/test/unittests/model/version